### DISCUSSION TITLE
Using woff2 fonts has unexpected behaviour, someone knows why?

### DISCUSSION THREAD

**Discussion by BsNoSi** [datenstrom/community#238] [Ask a question] [answered]
After setting up my css for a version 0.6.9 domain, I checked my results an noticed, that the fonts where not found in the `/system/themes/assets` folder. The browser does expect them in `media/themes/assets`.

After checking the `config.ini` I had an idea, why.

```
ThemeLocation: /media/themes/
AssetLocation: /media/themes/assets/
…
AssetDir: system/themes/assets/
```

But changing the first entries to `/system/themes/` did not help. I changed it back to `media` and created the expected folders and files in this folder. 

**Another site with same settings** behaves as expected. There the fonts are in `system/themes/assets/`, the fonts are found without errors thrown in browser.

Any idea, what happens there?

**Comment by markseuffert** [comment]
Yup, I can explain. Direct access to the system folders is not allowed.
Files needed for plugins and themes are mapped via media locations.

**Comment by BsNoSi** [comment]
Thanks for fast reply, but: why have I one domain, where I do not have to copy the files into the media folder and another, where I have to? *This* it the part, I do not understand .

**Comment by markseuffert** [comment]
Do you have an example. Which website is working as expected and which is not?

**Comment by BsNoSi** [comment]
I will send the two domains by mail.

**Comment by markseuffert** [comment, marked as answer]
Thanks Norbert, 

I think I know what is going on. Please update your core to the latest version from GitHub, e.g. `php yellow.php update force YellowCore`. This should support both woff and woff2 font formats, which you are using on your website. Support for woff2 has been added by @richi just a week ago. 

Let me know if this solves the mystery. 😄

**Comment by BsNoSi** [comment]
Yes – this was the reason. The *working site* was version 0.6.9, the *not working one* version 0.6.8. Problem was solved by checking the settings and update the recommended files.

THANK YOU, Mark. (→ can be closed)
